parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -911,6 +964,8 @@ public static class ReadBuilder {
     private boolean reuseContainers = false;
     private int maxRecordsPerBatch = 10000;
     private NameMapping nameMapping = null;
+    private ByteBuffer fileEncryptionKey = null;
+    private ByteBuffer fileAadPrefix = null;
```

Comment:
Like the write path, this should be `aadPrefix`.

Reply:
`WriteBuilder` also calls this field `fileAadPrefix` (line 140 in the class)